src/js/program/program-controller.js

```diff
@@ -266,7 +266,10 @@ class ProgramController extends Events {
             return;
         }
 
-        backgroundMediaController.mediaModel.attributes.mediaState = 'buffering';
+        const autoPause = model.get('autoPause');
+
+        // If background loading and using autoPause, we want to show the paused state after ads.
+        backgroundMediaController.mediaModel.attributes.mediaState = autoPause ? 'paused' : 'buffering';
```

Comment:
We do? I thought we nixed that idea.

If we so then just set `instream.noResume` to `true` and get rid of your changes in instream.

Reply:
The whole reason for the change in here is because the content would be paused after the ad but the player would be in a buffering state.

Reply:
But that's what `noResume` does. It prevents the attempt to play after an ad, so there is no buffering.

Reply:
If I revert the changes in program-controller to be back to `backgroundMediaController.mediaModel.attributes.mediaState = autoPause ? 'paused' : 'buffering';` and set `instream.noResume = true` in the controller, then the media state will still be set to buffering. Yeah, the video won't play but the player will still be showing the buffering state.